.github/workflows/github-actions.yml

```diff
@@ -25,7 +25,9 @@ concurrency:
 
 jobs:
   build:
-    runs-on: ubuntu-latest
+  
+    runs-on: ubuntu-22.04
```

Comment:
Remove extra newline and could you rename the Github Actions job and yml file to something meaning ('Build' or build.yml may work)

Reply:
Done

Reply:
Done

Reply:
Done